### PR TITLE
Inog14v report

### PR DESCRIPTION
**Pull request by modonovan**

### CODE REVIEW

**Comment by donalod** [comment]
@modonovan great stuff, I think I figured out the playlist link "https://www.youtube.com/embed/videoseries?list=PLWGYuk4mdX_9XFCuVvIMGfeJGProq5hzp" which should work, it's the embed specific format rather than the view?

**Comment by modonovan** [comment]
> @modonovan great stuff, I think I figured out the playlist link "https://www.youtube.com/embed/videoseries?list=PLWGYuk4mdX_9XFCuVvIMGfeJGProq5hzp" which should work, it's the embed specific format rather than the view?

Good spot - that's updated now and awaiting review.

**Comment by donalod** [comment]
@modonovan to get in to the workflow, are you ok to close out and resolve the changes requested by @cmsirbu by commenting/clicking on them above in https://github.com/inognet/inog/pull/126

File: pub/index.html [hunk 1]
```diff
@@ -142,6 +142,28 @@ <h5>uses Paypal</h5>
...
+          <br>Our three greater talks were:
+          </p>
+          <ul>
+            <li><strong><a href="https://www.linkedin.com/in/oliver-herms-7974b091/" target="_blank" class="text-primary">Oliver Herms</a> (<a href="https://twitter.com/taktv6">@taktv6)</a></strong>Source Routing on the Edge <a href="#" target="_blank" class="text-warning"><strong>SLIDES (TBD)</strong></a></li>
+            <li><strong><a href="https://www.linkedin.com/in/lsp42/" target="_blank" class="text-primary">David Gee</a> (<a href="https://twitter.com/_ipengineer">@_ipengineer)</a></strong>Naviating Automata Without a Map <a href="files/iNOG14v_davidgee.pdf" target="_blank" class="text-warning"><strong>SLIDES</strong></a></li>
+            <li><strong><a href="https://www.linkedin.com/in/mirceaulinic/" target="_blank" class="text-primary">Mircea Ulinic</a> (<a href="https://twitter.com/mirceaulinic">@mirceaulinic)</a></strong>Automating Networks Using Salt, Without Running Proxy Minions <a href="files/iNOG14v_mircea.pdf" target="_blank" class="text-warning"><strong>SLIDES</strong></a></li>
```
[the remark this answers is not in the document]
Comment (modonovan) [reply pub/index.html]: added spacing as requested.

File: pub/index.html [hunk 2]
```diff
@@ -142,6 +142,28 @@ <h5>uses Paypal</h5>
...
+           </ul>
+           <p>Special thanks this time around must go to <strong>all our attendees</strong> who embraced the virtual meeting with us. We had a large number of our usual attendees but because we ran our meeting entirely online we also had a big number of new attendees from all over the world which was fantastic to see. Our "virtual" social took place following the talks as is in keeping with our usual format but this time using Jitsi Meet. Watch this space for a similar format for iNOG::15v next month</p>
```
[the remark this answers is not in the document]
Comment (modonovan) [reply pub/index.html]: Completed as requested

File: pub/index.html [hunk 3]
```diff
@@ -142,6 +142,28 @@ <h5>uses Paypal</h5>
...
+       <!-- iNOG::14v-->
+      <div class="row">
+        <div class="col-lg-6">
+          <h3>iNOG::14v<small> 2020.04.17</small></h3>
+          <p class="h5">Maiden voyage into Virtual Meeting space at a time of #COVID-19!</p>
+          <p>Another iNOG, another new venue, but with a difference - our homes! A new departure saw us run our meeting entirely remotely.
+          <br>Our three greater talks were:
+          </p>
+          <ul>
+            <li><strong><a href="https://www.linkedin.com/in/oliver-herms-7974b091/" target="_blank" class="text-primary">Oliver Herms</a> (<a href="https://twitter.com/taktv6">@taktv6)</a></strong>Source Routing on the Edge <a href="#" target="_blank" class="text-warning"><strong>SLIDES (TBD)</strong></a></li>
```
[the remark this answers is not in the document]
Comment (modonovan) [reply pub/index.html]: done

**Comment by modonovan** [comment]
In concluding the updates I did a final sanity check and noticed another grammar mistake whereby the word "greater" was used instead of "great" - this is now fixed.

**Pull request merged**